README.md

```diff
@@ -1,3 +1,7 @@
 # demo-rip
 
 Decided to open a new repository, I'm tired of Git for real.
+
+## local development
+
+1. Open Index.html
```

Comment:
Hopefully, we do this some other time. 

They say we should not be afraid to break our code.